### PR TITLE
Update to aws-perfscale-qe profile 

### PR DESCRIPTION
**Pull request by svetsa-rh**
Update all tests to use aws-perfscale-qe profile and update tests missing some env.

### CODE REVIEW

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-nightly-perfscale-rosa-multiaz-cluster-density-v2-3nodes

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-cluster-density-v2-3nodes

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-cluster-density-v2-3nodes

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-cluster-density-v2-3nodes

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-nightly-perfscale-rosa-multiaz-ingress-perf-3nodes-periodic

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-cluster-density-v2-24nodes-periodic

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-stable-prod.yaml [hunk 1]
```diff
@@ -57,13 +57,12 @@ tests:
     - chain: openshift-qe-cluster-density-v2
     workflow: rosa-aws-sts-hypershift
 - always_run: false
-  as: perfscale-rosa-multiaz-ingress-perf-3nodes
+  as: perfscale-rosa-multiaz-ingress-perf-defaultnodes
```
[the remark this answers is not in the document]
Comment (krishvoor) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-stable-prod.yaml:60]: @svetsa-rh I am good with this change in general.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly.yaml [hunk 2]
```diff
@@ -78,10 +78,14 @@ tests:
...
   steps:
-    cluster_profile: aws-qe
+    cluster_profile: aws-perfscale-qe
     env:
       CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
+      ENABLE_AUTOSCALING: "false"
+      MULTI_AZ: "true"
       OPENSHIFT_VERSION: "4.12"
+      REPLICAS: "3"
```
[the remark this answers is not in the document]
Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly.yaml:88]: @krishvoor What about this one? Should I remove REPLICAS and MULTIAZ and rename test?

[the remark this answers is not in the document]
Comment (krishvoor) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly.yaml:88]: With `--multi_az` the minimum worker node count is 3.
And this change should not have ideally effected, but can we try to flip it to this:

```
    cluster_profile: aws-perfscale-qe
    env:
      CHANNEL_GROUP: nightly
      COMPUTE_MACHINE_TYPE: m5.2xlarge
      ENABLE_AUTOSCALING: "false"
      MULTI_AZ: "true"
      OPENSHIFT_VERSION: "4.12"
```

[the remark this answers is not in the document]
Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly.yaml:88]: I don't see default value being set for REPLICAS in `rosa-aws-sts` workflow, so we might end up with other errors or prompting us to set `REPLICAS: "3"`

[the remark this answers is not in the document]
Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly.yaml:88]: I checked again and see that the default value for REPLICAS is 2 [here](https://github.com/svetsa-rh/release/blob/2088dbfe3009eb31368d5aa741c155cc69a91628/ci-operator/step-registry/rosa/sts/cluster/provision/rosa-sts-cluster-provision-ref.yaml#L30), so if we set ENABLE_AUTOSCALING: "false" and MULTI_AZ: "true", we will definitely see error that we need to set 3 replicas.

[the remark this answers is not in the document]
Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly.yaml:88]: @krishvoor @qiliRedHat Reverted this change for now to proceed merging this PR so other tests can start running with new profile. I'll open a new PR to address this issue.

**Comment by svetsa-rh** [comment]
@qiliRedHat Can you ptal? Here is a [passing run of cluster-density-v2](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/41554/rehearse-41554-periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-cluster-density-v2-24nodes-periodic/1683957538699087872)
I'll open separate PR for updating ingress test with proper env variables.

**Comment by svetsa-rh** [comment]
@qiliRedHat Can you ptal? Here is a passing run [cluster-density-v2](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/41554/rehearse-41554-periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-cluster-density-v2-24nodes-periodic/1683957538699087872) 
I reverted changes for 4.12 ingress tests and will create separate PR for it.

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-nightly-perfscale-rosa-multiaz-ingress-perf-3nodes-periodic

**Comment by svetsa-rh** [comment]
Current status of PR: Needs necessary labels for merging. 
Classic Rosa cluster-density-v2 is passing
HCP has Prow infra issues so failing but not related to our changes in this PR 
and 
ingress test on Classic Rosa is failing, but also unrelated. More discussion at https://redhat-internal.slack.com/archives/GTHU29259/p1690348181049899?thread_ts=1690303212.904219&cid=GTHU29259

**Pull request merged**